Error when running npm install

There is a problem with a dependency:
npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/esprima-six

confirmed, looks like esprima-six was unplublished, now just to figure out
what required it...

On Fri, Nov 20, 2015 at 5:49 AM Cedrik Vanderhaegen <
notifications@github.com> wrote:

> There is a problem with a dependency:
> npm ERR! 404 Registry returned 404 for GET on
> https://registry.npmjs.org/esprima-six
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/proj4js/proj4js/issues/163>.
>

I still have the error, I just tried with a clean install. 
I do git clone into an empty dir and run npm install in that dir. This is the error:
```
npm ERR! Linux 4.2.0-27-generic
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install"
npm ERR! node v0.12.9
npm ERR! npm  v2.14.9
npm ERR! code E404

npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/esprima-six
npm ERR! 404 
npm ERR! 404 'esprima-six' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'syntax-error'
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/cedrik/tests/proj4/proj4js/npm-debug.log
```
I'm working on ubuntu 15.10

--- a/README.md
+++ b/README.md
@@ -89,7 +89,7 @@ proj4.defs([
 ]);
 ```
 
-you can then do 
+you can then do
 
 ```js
 proj4('EPSG:4326');
@@ -112,7 +112,7 @@ proj4.defs can also be used to define a named alias:
 
 ```javascript
 proj4.defs('urn:x-ogc:def:crs:EPSG:4326', proj4.defs('EPSG:4326'));
-``` 
+```
 
 ## Developing
 to set up build tools make sure you have node and grunt-cli installed and then run `npm install`
@@ -120,7 +120,7 @@ to set up build tools make sure you have node and grunt-cli installed and then r
 to do the complete build and browser tests run
 
 ```bash
-grunt
+node_modules/.bin/grunt
 ```
 
 to run node tests run
@@ -132,22 +132,22 @@ npm test
 to run node tests with coverage run
 
 ```bash
-node test --coverage
+npm test --coverage
 ```
 
-to create a build with only default projections (latlon and Mercator) run 
+to create a build with only default projections (latlon and Mercator) run
 
 ```bash
-grunt build
+node_modules/.bin/grunt build
 ```
 
 to create a build with only custom projections include a comma separated list of projections codes (the file name in 'lib/projections' without the '.js') after a colon, e.g.
 
 ```bash
-grunt build:tmerc
+node_modules/.bin/grunt build:tmerc
 #includes transverse Mercator
-grunt build:lcc
+node_modules/.bin/grunt build:lcc
 #includes lambert conformal conic
-grunt build:omerc,moll
+node_modules/.bin/grunt build:omerc,moll
 #includes oblique Mercator and Mollweide
 ```

--- a/lib/datum.js
+++ b/lib/datum.js
@@ -20,7 +20,7 @@ var datum = function(proj) {
   }
 
   if (proj.datum_params) {
-    this.datum_params = proj.datum_params.map(parseFloat); 
+    this.datum_params = proj.datum_params.map(parseFloat);
     if (this.datum_params[0] !== 0 || this.datum_params[1] !== 0 || this.datum_params[2] !== 0) {
       this.datum_type = PJD_3PARAM;
     }
@@ -34,7 +34,7 @@ var datum = function(proj) {
       }
     }
   }
-  
+
   // DGR 2011-03-21 : nadgrids support
   this.datum_type = proj.grids ? PJD_GRIDSHIFT : this.datum_type;
 

--- a/package.json
+++ b/package.json
@@ -33,9 +33,9 @@
     "chai": "~1.8.1",
     "mocha": "~1.17.1",
     "grunt-mocha-phantomjs": "~0.4.0",
-    "browserify": "~3.24.5",
-    "grunt-browserify": "~1.3.0",
-    "grunt-contrib-uglify": "~0.3.2",
+    "browserify": "~12.0.1",
+    "grunt-browserify": "~4.0.1",
+    "grunt-contrib-uglify": "~0.11.1",
     "curl": "git://github.com/cujojs/curl.git",
     "istanbul": "~0.2.4",
     "tin": "~0.4.0"

